src/jit/emitxarch.cpp

```diff
@@ -2290,12 +2395,17 @@ inline UNATIVE_OFFSET emitter::emitInsSizeAM(instrDesc* id, code_t code, int val
     {
         valSize = sizeof(char);
     }
+    else
+    {
+        assert(!IsSSEOrAVXInstruction(ins));
```

Comment:
These should catch any remaining places where we are incorrectly passing in an unsigned value > 127 to `ival`